public/getting-started/index.html

```diff
@@ -0,0 +1,260 @@
+          </li>
+        </ol>
+      </li>
+      <li><b>Update the home page link for your site</b>
+        <ol>
+          <li>Click the <code>package.json</code> file in the root directory of
+            the repository.
+             <div><a href="package.png" target="_blank"><img src="package.png" width="800px"></a></div>
+          </li>
+          <li>Click the <code>edit</code> icon.
+             <div><a href="edit.png" target="_blank"><img src="edit.png" width="800px"></a></div>
+          </li>
+          <li>Replace <code>perseids-publications.github.io</code> with, e.g.,
+            <code>yourgithubaccount.github.io</code>.
+            That is, if your GitHub user name is "janedoe" you would change
+            <code>https://perseids-publications.github.io/treebank-template/</code> to
+            <code>https://janedoe.github.io/treebank-template/</code>
+             <div><a href="homepage.png" target="_blank"><img src="homepage.png" width="800px"></a></div>
+          </li>
+          <li>Scroll to the bottom of the page to <code>Commit Changes</code>
+            and enter a message to describe your
+            change to this file. (e.g. something like "Updated homepage link")
+          </li>
+          <li>Make sure the option <code>Commit directly to the master branch</code>
+            is selected
+             <div><a href="commit.png" target="_blank"><img src="commit.png" width="800px"></a></div>
+          </li>
+          <li>Click <code>Commit Changes</code></li>
+        </ol>
+      </li>
+      <li><b>Setup GitHub Pages</b>
+        <ol>
+          <li>Click the <code>Settings</code> button
+             <div><a href="settings.png" target="_blank"><img src="settings.png" width="800px"></a></div>
+           </li>
+           <li>Scroll down to the <code>GitHub Pages</code> section of the page.</li>
+          <li>Select <code>gh-pages branch</code> for the source from the
+            dropdown. <b>NOTE:</b> it is important that you select the
+            <code>gh-pages branch</code> option from the dropdown, <b>even if
+              its preselected</b>. It seems that the process of actually selecting it is
+              necessary to trigger GitHub to actually acknowledge the setting.
+              After this, you should see a notice <code>Your site is ready to be
+              published at https://youraccount.github.io/treebank-template</code>
+             <div><a href="ghpages.png" target="_blank"><img src="ghpages.png" width="800px"></a></div>
+          </li>
+        </ol>
+      </li>
+      <li><b>Update Site Details</b>
+        <ol>
+          <li>Go back to the <code>Code</code> tag and navigate to
+            the <code>src</code> directory. Click on the <code>config.json</code>
+            file.
+             <div><a href="config.png" target="_blank"><img src="config.png" width="800px"></a></div>
+          </li>
+          <li>Edit the file and update the following data fields:
```

Comment:
I really like that the examples use the built in GitHub editor. It's much easier for people unfamiliar with Git and the command line to use the web interface.

Reply:
Thanks. Ultimately, I think they will have to get at least a little familiar with GitHub to really use the template, but this hopefully makes it easier to get started.